packages/material-react-table/src/inputs/MRT_FilterTextField.tsx

```diff
@@ -425,10 +428,8 @@ export const MRT_FilterTextField = <TData extends MRT_RowData>({
                     </Box>
                   )
               : undefined,
+            ...commonTextFieldProps.SelectProps
           }}
-          onChange={handleTextFieldChange}
```

Comment:
Pretty sure the onChange needs to stay down here

Reply:
Do you mean that user should not be able to overwrite it with `{...commonTextFieldProps}`?

Like
`
{...commonTextFieldProps}
onChange={handleTextFieldChange}
`

Reply:
yes, I think it would be a breaking change. The `handleTextFieldChange` function already calls the onChange passed in by the dev. Making this change would remove the setState functionality.